samples/MvvmSampleUwp/MvvmSampleUwp/MvvmSampleUwp.csproj

```diff
@@ -256,14 +256,17 @@
     </ApplicationDefinition>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection">
+      <Version>3.1.9</Version>
+    </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
       <Version>6.2.10</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit">
-      <Version>7.0.0-preview2</Version>
+      <Version>7.0.0-preview3</Version>
```

Comment:
Make sure to align the same toolkit versions here across all packages used, sometimes weird things can happen otherwise...

Reply:
I left the Preview2 for the controls packages due to that issue with the `MarkdownTextBlock`. I could bump that now but I was just worried about making larger changes just 6 hours before the talk 😅

Reply:
ah, that should be resolved by using the main CI feed. We've tested it. :)